js/liquid.js

```diff
@@ -279,7 +279,7 @@ module.exports = class liquid extends Exchange {
             let total = parseFloat (balance['balance']);
             let account = {
                 'free': total,
-                'used': undefined,
+                'used': 0.0,
```

Comment:
We usually don't set it, if we don't know it.

Reply:
Let me know your thoughts on how we should proceed with this.

Reply:
This is rather new to me, the other exchanges I use, e.g. [kraken set it to 0.0](https://github.com/ccxt/ccxt/blob/f8a836a857860ee6e121eab78bcc04865e0e5974/js/kraken.js#L825).
I've introduced a bug in my code because I expected used to be numerical. I think the less dangerous way is to set it to 0.0 but add a comment in the docs, that some/most exchanges don't support used/free amounts.

Reply:
something like exchange.has['used_amounts'] could also be added